website/docs/r/organization_run_task_global_settings.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization_run_task_global_settings"
+description: |-
+  Manages Run tasks global settings.
+---
+
+# tfe_organization_run_task_global_settings
+
+[Run tasks](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/run-tasks) allow HCP Terraform to interact with external systems at specific points in the HCP Terraform run lifecycle. Run tasks are reusable configurations that you can attach to any workspace in an organization.
+
+The tfe_organization_run_task_global_settings resource creates, updates and destroys the [global settings](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/run-tasks#global-run-tasks) for an [Organization Run task](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/run-tasks#creating-a-run-task). Your organization must have the `global-run-task` [entitlement](https://developer.hashicorp.com/terraform/cloud-docs/api-docs#feature-entitlements) to use global run tasks.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_organization_run_task_global_settings" "example" {
+		task_id = tfe_organization_run_task.example.id
+		enabled           = false
+		enforcement_level = "advisory"
+		stages            = ["pre_plan", "post_plan"]
+}
```

Comment:
Can you provide the whole example where `tfe_organization_run_task.example` is included as well?

Does `enabled` need to be an argument in both resources?

Reply:
Added the missing example, and set enabled to true in both to avoid confusion.

> Does enabled need to be an argument in both resources?
Yes.